modules/food-menu/fancy-menu.js

```diff
@@ -94,6 +96,13 @@ export class FancyMenu extends React.Component<Props, State> {
 		})
 	}
 
+	onPressRow(item: MenuItem, icons: MasterCorIconMapType) {
+		this.props.navigation.navigate('MenuItemDetailView', {
```

Comment:
Use object shorthand here, please – `{item, icons}`. We'll get used to it if we use it.

Reply:
Did I address this or did I modify the incorrect line?